README.md

```diff
@@ -18,50 +18,51 @@ Please do not edit anything outside the answers sections.
 
 # Q1: Mr Fussy Builds a Path
 
-# Q1
+Grammars Assignment (50 points)
 
-Mr Fussy wants to build a path using a row of large square pavers. His path is
-one paver wide and an odd number of pavers long. Pavers come in three colors:
-red, green, blue.
+This file contains two questions. The first has 2 parts, the second has 7.
+
+For all but questions 2.3 and 2.5 you'll answer questions by editing this file. Your answers will appear in the Ax.y section after each question, replacing the placeholders.
 
-Mr Fussy has a rule: the sequence of colors must be symmetrical across the
-length of the path: the last paver must be the same color as the first, the
-second to last the same color as the second, and so on.
+For question 2.3, you'll need to add an image file to the repository (.png or .pdf) and put its file name as the answer.
 
-This is a valid path:  rgbgr
-So is this: rrr
+For question 2.5, you'll write one or more source code files and add them to the repository. You will then show the command used to run the files in the answer. Do not upload any binary files for this section.
 
-This is not: rgb
-This is not: rggr    (it must be an odd length)
+Please do not edit anything outside the answers sections.
 
-## Q1.1  (5 points)
+Q1: Mr Fussy Builds a Path
 
-Write a Chomsky type 2 grammar that describes any valid path containing at
-least one paver. Use `S` as the start state, and `r`, `g`, `b` as terminals that
-represent the tiles.
+Q1
 
-## A1.1
+Mr Fussy wants to build a path using a row of large square pavers. His path is one paver wide and an odd number of pavers long. Pavers come in three colors: red, green, blue.
 
-«replace this with your answer»
+Mr Fussy has a rule: the sequence of colors must be symmetrical across the length of the path: the last paver must be the same color as the first, the second to last the same color as the second, and so on.
 
+This is a valid path: rgbgr So is this: rrr
 
-## Q1.2  (1 point for the O() answer, 2 for the sentence)
+This is not: rgb This is not: rggr (it must be an odd length)
 
-Using big-O notation, what is the likely memory requirement for a parser that
-can validate a particular path configuration, where `n` is the number of tiles?
-In one sentence, explain why.
+Q1.1 (5 points)
 
-## A1.2
+Write a Chomsky type 2 grammar that describes any valid path containing at least one paver. Use S as the start state, and r, g, b as terminals that represent the tiles.
 
-«replace this with your answer»
+A1.1
 
+S -> r S r S -> g S g S -> b S b S -> r S -> g S -> b
```

Comment:
I assume you meant
~~~
S -> r S r 
S -> g S g 
S -> b S b 
S -> r 
S -> g 
S -> b
~~~

which is correct